tests/nowdoc/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -356,6 +389,40 @@ EOF
     .
     $var;
 
+$str = <<<'EOD'
+
+EOD;
```

Comment:
Bug in parser https://github.com/glayzzle/php-parser/issues/191, but not critical